src/Windows/R/Components/Impl/Plots/Implementation/View/RPlotHistoryControl.xaml.cs

```diff
@@ -61,24 +62,36 @@ private void ActivatePlot(IRPlotHistoryEntryViewModel entry)
         }
 
         private void HistoryListView_MouseLeave(object sender, MouseEventArgs e) => _dragSurface.MouseLeave(e);
-        private void HistoryListView_PreviewMouseDown(object sender, MouseButtonEventArgs e) => _dragSurface.MouseDown(e);
 
-        private void HistoryListView_MouseRightButtonUp(object sender, MouseButtonEventArgs e) {
+        private void HistoryListView_PreviewMouseLeftButtonDown(object sender, MouseButtonEventArgs e) {
+            _dragSurface.MouseDown(e);
+            var count = SelectedPlots.Count();
+            if (count == 0) {
+                return; // Allow selection
+            }
+
+            var control = Keyboard.IsKeyDown(Key.LeftCtrl) || Keyboard.IsKeyDown(Key.RightCtrl);
+            if (!control) {
+                // Regular click clear selection then reselects
+                HistoryListView.SelectedItems.Clear();
+            }
+        }
+
+        private void HistoryListView_PreviewMouseLeftButtonUp(object sender, MouseButtonEventArgs e) => e.Handled = true;
```

Comment:
List view has this odd 'I am a checkbox' behavior. Click selects element and then more elements (without Control pressed). One of the fixes is to make sure plain click selects primary element and ctrl+click adds/removes. So LBUp behavior is overridden just in case.